ql/src/java/org/apache/hadoop/hive/ql/stats/StatsUtils.java

```diff
@@ -894,6 +878,40 @@ public static ColStatistics getColStatistics(ColumnStatisticsObj cso, String col
     return cs;
   }
 
+  public static void fillColumnStatisticsData(ColumnStatisticsData data, ColStatistics cs,
+      String colType) throws MetaException {
+    ColStatistics.Range r = cs.getRange();
+    Object lowValue = r != null ? r.minValue : null;
+    Object highValue = r != null ? r.maxValue : null;
+    StatObjectConverter.fillColumnStatisticsData(colType, data, lowValue, highValue,
+        cs.getNumNulls(), cs.getCountDistint(), cs.getBitVectors(), cs.getHistogram(),
```

Comment:
binary bitVectors & histogram are not particularly useful in describe table, so maybe keep nulls